plugins/CoreUpdater/Updater.php

```diff
@@ -298,11 +298,6 @@ private function installNewFiles($extractedArchiveDirectory)
             $model->removeGoneFiles($extractedArchiveDirectory, PIWIK_DOCUMENT_ROOT);
         }
 
-        // Config files may be user (account) specific
-        if (PIWIK_INCLUDE_PATH !== PIWIK_USER_PATH) {
```

Comment:
no longer needed to copy the global config files over as there might not be write permission anyway and it's not needed. They should be always in document root.